src/frontend/ai-reviewer-admin/src/axiosConfig.js

```diff
@@ -0,0 +1,7 @@
+import axios from "axios";
+
+const api = axios.create({
+  baseURL: "http://localhost:8090/",
```

Comment:
This will become a env var in the future I assume.

Reply:
Absolutely. Doing that now in a subsequent PR - expose params to docker-compose.

Reply:
Awesome!